.docker/Dockerfile.local

```diff
@@ -1,20 +1,55 @@
-FROM node:8
+FROM ubuntu:16.04 as builder
 
-ADD . /app
+RUN apt update && apt install curl git bzip2 g++ build-essential python -y
 
-ENV RC_VERSION=0.57.0-designpreview \
-    DEPLOY_METHOD=docker \
-    NODE_ENV=production \
-    PORT=3000 \
-    ROOT_URL=http://localhost:3000
+# meteor installer doesn't work with the default tar binary
+RUN apt-get install -y bsdtar \
+    && cp $(which tar) $(which tar)~ \
+    && ln -sf $(which bsdtar) $(which tar) \
+    && curl "https://install.meteor.com/?release=1.6.0.1" \
+    | sed 's/VERBOSITY="--silent"/VERBOSITY="--progress-bar"/' \
+    | sh \
+    && mv $(which tar)~ $(which tar)
+
+COPY . /app
+
+# Add Chatpal which is a submodule
+WORKDIR /app
+RUN meteor npm i \
+    && meteor npm run postinstall \
+    && set +e \
+    && meteor add rocketchat:lib --allow-superuser \
+    && set -e \
+    && meteor build --allow-superuser --server-only --headless --directory /tmp/build
+
+FROM assistify/chat-base:latest
```

Comment:
I've seen this image is basically copy-paste from the RC base - except
`ENV DEBIAN_FRONTEND noninteractive` and an `apt-get update`.
Can you explain some motivations about that?
Also note, that using the ENV is discouraged in favor of inline args (https://serverfault.com/a/797318)

Reply:
is used as non interactive to suppress the prompt while installing package using `apt-get` command. However, this was resolved using `-y` inline `apt-get` command. So the `ENV DEBIAN_FRONTEND` is not required.